src/js/components/Data/stories/Simple.js

```diff
@@ -19,7 +19,20 @@ export const Simple = () => (
       status="info"
       message="Data is in 'beta'. The API surface is subject to change."
     />
-    <Data data={DATA} toolbar>
+    <Data
```

Comment:
If I select a view and then attempt to make changes to the filters, my changes are not applied.

Also searching doesn't work while I have a view selected.

https://user-images.githubusercontent.com/54560994/209198305-9adb9048-6f08-46f4-ad14-bb6dc4f26386.mp4

Reply:
Thanks for catching. These should be fixed now.